Tools/CISupport/ews-build/steps_unittest.py

```diff
@@ -5656,5 +5658,154 @@ def test_failure_merged(self):
         return self.runStep()
 
 
+class TestAddReviewerToCommitMessage(BuildStepMixinAdditions, unittest.TestCase):
+    ENV = dict(
+        GIT_COMMITTER_NAME='WebKit Committer',
+        GIT_COMMITTER_EMAIL='committer@webkit.org',
+        FILTER_BRANCH_SQUELCH_WARNING='1',
+    )
+
+    def setUp(self):
+        self.longMessage = True
+
+        def mock_load_contributors(*args, **kwargs):
```

Comment:
Let's try to avoid duplicating this (it's also in TestValidateCommitterAndReviewer)